Commands/notify.py

```diff
@@ -138,7 +142,8 @@ async def notify(self, ctx: commands.Context, list_name: str, *, message: typing
             )
 
         await ctx.channel.send(embed=embed)
-        await ctx.send(users_str)
+        for users_str in user_messages:
+            await ctx.send(users_str)
```

Comment:
To solve #96 you could run rstrip on the `users_str` to cut of the `, ` or you could slice those 2 chars away